tests/unit/test_configuration.py

```diff
@@ -34,6 +30,68 @@ def test_get_default_configuration(mocker):
     mock.assert_called_once_with(layers=["baseline", "default", "user"])
 
 
+def test_default_configuration_structure():
+    """Test that the default configuration structure is correct"""
+    config = get_configuration()
+    # Check forms
+    assert set(f.name for f in FORMS) == set(config.keys())
+    for form in FORMS:
+        # Check row noise
+        for row_noise in form.row_noise_types:
+            config_probability = config[form.name].row_noise[row_noise.name].probability
+            default_probability = (
+                DEFAULT_NOISE_VALUES.get(form.name, {})
+                .get("row_noise", {})
+                .get(row_noise.name, {})
+                .get("probability", {})
+            )
+            if default_probability:
```

Comment:
What if the default overrides the value to 0 or some other falsey value? You should specifically test that it is an empty dictionary or better yet change the last `get` call to `.get("probability", None)` and check `if default_probability is None:`

Reply:
Hmm, I don't think setting the last get return to None works either since it wouldn't distinguish between the default dictionary missing that item vs setting it to None. 

I'll just return "no default" to stay away from any ambiguity.